backend/src/main/scala/bloop/logging/BloopLogger.scala

```diff
@@ -50,6 +52,14 @@ class BloopLogger(override val name: String, out: PrintStream, err: PrintStream)
     coreLogger.addAppender(errAppender)
   }
 
+  private[this] def removeAllAppenders(): Unit = BloopLogger.synchronized {
```

Comment:
This is starting to look quite similar to what we had before 😛

Reply:
Yeah that's because in the initial version of this PR, I'd create a new logger for every session, but I reverted that later...